webpack.config.server.js

```diff
@@ -6,10 +6,14 @@ module.exports = ({ resolvePath, START_DEV_SERVER }) => {
   const serverConfig = {
     target: 'node', // compile for server environment
     entry: START_DEV_SERVER ? ['webpack/hot/poll?100', './src'] : ['./src'],
+    devtool: false,
```

Comment:
Whys is this needed?

Reply:
> This option controls if and how source maps are generated.
https://webpack.js.org/configuration/devtool/
Setting it to false stops producing a map file.